std/dynamic_library.zig

```diff
@@ -154,3 +163,27 @@ fn checkver(def_arg: *elf.Verdef, vsym_arg: i32, vername: []const u8, strings: [
     const aux = @intToPtr(*elf.Verdaux, @ptrToInt(def) + def.vd_aux);
     return mem.eql(u8, vername, cstr.toSliceConst(strings + aux.vda_name));
 }
+
+pub const WindowsDynLib = struct {
```

Comment:
This code should probably be merged with https://github.com/ziglang/zig/blob/af3263d7a87c922cbcb047944ee94123c69cfac1/std/os/windows/util.zig#L191

(we can keep the best API but let's not have 2 ways to do it)

Reply:
wow. i really need to actually look through all the code before i start "adding" features. completely missed these!

Reply:
It's not really your fault - we don't have auto generated docs with a search function yet.